website/docs/docs/deploy/continuous-integration.md

```diff
@@ -0,0 +1,62 @@
+
+dbt Cloud builds and tests the models affected by the code change in a temporary schema, unique to the PR. This process ensures that the code builds without error and that it matches the expectations as defined by the project's dbt tests. The unique schema name follows the naming convention `dbt_cloud_pr_<job_id>_<pr_id>` (for example, `dbt_cloud_pr_1862_1704`) and can be found in the run details for the given run, as shown in the following image:
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/using_ci_dbt_cloud.png" width="90%"title="Viewing the temporary schema name for a run triggered by a PR"/>
+
+When the Slim CI run completes, you can view the run status directly from within the pull request. dbt Cloud updates the pull request in GitHub, GitLab, or Azure DevOps with a status message indicating the results of the run. The status message states whether the models and tests ran successfully or not. 
+
+dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has database or schema customization using the [generate_database_name](/docs/build/custom-databases#generate_database_name) or [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Temp PR schema limitations](#temp-pr-schema-limitations).
+
+## Differences between Slim CI jobs and other deployment jobs
+
+The [dbt Cloud scheduler](/docs/deploy/job-scheduler) executes Slim CI jobs differently from other deployment jobs in these important ways:
+
+- **Concurrent CI checks** &mdash; Slim CI runs triggered by the same dbt Cloud Slim CI job execute concurrently (in parallel), when appropriate
+- **Smart cancellation of stale builds** &mdash; Automatically cancels stale, in-flight Slim CI runs when there are new commits to the PR
+- **Run slot treatment** &mdash; Slim CI runs don't consume a run slot
+
+:::tip Join our beta
+This functionality is currently in beta. If you're interested in joining our beta, please [contact us](mailto:support@getdbt.com) for access.
+
+:::
+
+### Concurrent CI checks
+
+When you have teammates collaborating on the same dbt project creating pull requests on the same dbt repository, the same Slim CI job will get triggered. Since each run builds into a dedicated, temporary schema that’s tied to the pull request, dbt Cloud can safely execute Slim CI runs concurrently instead of sequentially (differing from what is done with deployment dbt Cloud jobs). Because no one needs to wait for a Slim CI run to finish before another one can start, with concurrent CI checks, your whole team can test and integrate dbt code faster.
```

Comment:
```suggestion
When you have teammates collaborating on the same dbt project creating pull requests on the same dbt repository, the same Slim CI job will get triggered. Since each run builds into a dedicated, temporary schema that’s tied to the pull request, dbt Cloud can safely execute Slim CI runs _concurrently_ instead of _sequentially_ (differing from what is done with deployment dbt Cloud jobs). Because no one needs to wait for a Slim CI run to finish before another one can start, with concurrent CI checks, your whole team can test and integrate dbt code faster.
```